redis-persistence/src/main/java/com/netflix/conductor/dao/DynoProxyProvider.java

```diff
@@ -0,0 +1,24 @@
+package com.netflix.conductor.dao;
+
+
+import com.netflix.conductor.dao.dynomite.DynoProxy;
+
+import javax.inject.Inject;
+import javax.inject.Provider;
+
+import redis.clients.jedis.JedisCommands;
+
```

Comment:
@Singleton?

Reply:
I think this should be covered by that fact that `DynoProxy` itself is a singleton.  Unless you think the `Provider` itself should be a `@Singleton` for some reason.

Reply:
I take it back, since `new` is being called directly think that breaks this.  But it does make me think that these two providers may be unnecessary.